xedocs/schemas/corrections/bayes_model.py

```diff
@@ -0,0 +1,9 @@
+"""
+
+"""
+
+from .base_corrections import TimeIntervalCorrection
+
+
+class BayesModel(TimeIntervalCorrection):
+    _ALIAS = "bayes_models"
```

Comment:
Dont you want to add any fields? what type is the value?
Does it reference a file in our resource database? if so it should inherit from the BaseResourceReference class no?